packages/db/src/meta/interface.ts

```diff
@@ -88,13 +81,9 @@ export const forAttachAndSchema = <C extends Collections>(options: {
     };
   };
 
-  const serve = (config: TruffleConfig | ConnectOptions<C> | undefined) => {
-    // TODO: get rid of @truffle/config stuff and rename field to "directory"
+  const serve = (config: ConnectOptions<C> | undefined) => {
```

Comment:
(Same as above)

```suggestion
  const serve = (config?: ConnectOptions<C>) => {
```

Reply:
(Same as above for the other above)

```suggestion
  const serve = (options: ConnectOptions<C> | undefined) => {
```

Reply:
Ooh, I committed this but then realized that `options` is the argument passed to the parent function so no-can-do.